starlight/src/content/docs/blog/starlight-sidebar-whitespace.md

```
@@ -0,0 +1,146 @@
+```css {3}
+// src/styles/custom.css
+sl-sidebar-state-persist ul.top-level > li > a {
+  font-size: var(--sl-text-sm);
+}
+```
+
+![Starlight sidebar with smaller font size](../../../assets/sidebar-css/font-size.png)
+
+## Recommendations
+
+Summing everything up, I recommend that you apply some mix of some of the mentioned customization options above, which you can conveniently see here in one single `.css` file. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)